lib/bolt/inventory.rb

```diff
@@ -108,6 +109,15 @@ def vars(target)
       @target_vars[target.name]
     end
 
+    def add_facts(target, new_facts = {})
+      # TODO: Should this warn if facts is empty?
```

Comment:
If `new_facts` is empty? I'm on the fence. I can imagine scenarios where you might update facts, but expect that some are empty and not want to have to separately filter them. But also that empty facts could be a warning of something going wrong.

I guess let's warn for now.